content/source/docs/cloud/integrations/run-tasks/index.html.md

```diff
@@ -13,13 +13,13 @@ In addition to using existing technology partners integrations, HashiCorp Terraf
 
 ## Prerequisites
 
-To build a custom integration, you must have a server capable of receiving requests from Terraform Cloud and responding with a status update to a supplied callback URL. When creating an event hook, you supply an endpoint url to receive the hook. We send a test POST to the supplied URL, and it must respond with a 200 for the event hook to be created.
+To build a custom integration, you must have a server capable of receiving requests from Terraform Cloud and responding with a status update to a supplied callback URL. When creating a run task, you supply an endpoint url to receive the hook. We send a test POST to the supplied URL, and it must respond with a 200 for the run task to be created.
 
 This feature relies heavily on the proper parsing of [plan JSON output](../../../internals/json-format.html). When sending this output to an external system, be certain that system can properly interpret the information provided.
 
 ## Integration Details
 
-When a run reaches the pre-apply phase and an event hook is triggered, the supplied URL will receive details about the run in a payload similar to the one below. The server receiving the event hook should respond `200 OK`, or Terraform will retry to trigger the hook.
+When a run reaches the pre-apply phase and a run task is triggered, the supplied URL will receive details about the run in a payload similar to the one below. The server receiving the run task should respond `200 OK`, or Terraform will retry to trigger the run task.
```

Comment:
This section explicitly calls out the "pre-apply" phase, it's my understandting that this no longer exists, so perhaps we should reword this and use simpler terminology to explain the concept of when a run task will trigger.